### PR TITLE
feat: Print fizzbuzz if i multiple of 15.

### LINKED ISSUE #1: team_sh
## Description

python활용 구현

## Tasks
- [ ] % if
- [ ] % 15
- [ ] % 5
- [ ] % 3
- [x] #8

## References

### PR DESCRIPTION
**Pull request by Jodory**
close #1 Resolve printing fizzbuzz if i multiple of 15.

### CODE REVIEW

File: fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+for i in range(1, 16 + 1):
+    if i % 15 == 0:
+        print('fizzbuzz')
+    elif i % 3 == 0:
+        print('fizz')
+    elif i % 5 == 0:
+        print('buzz')
+    else:
+        print(i)
```
Comment (2xel) [inline fizzbuzz.py:5]: 멋진 코드입니다.

**Comment by Jodory** [comment]
안녕하세요

File: fizzbuzz.py (hunk 1 above)
Comment (Jodory) [reply fizzbuzz.py:5]: 감사합니다~~

**Review by Jodory — COMMENTED** [review]
FIx: print fizzbuzz.py

**Pull request merged**